conda_smithy/lint_recipe.py

```diff
@@ -149,6 +149,25 @@ def lintify(meta, recipe_dir=None):
     except RuntimeError as e:
         lints.append(str(e))
 
+    # 13: If cython is a build-dependency then python should be a run-time
+    # dependency. This ensures that packages will be marked correctly by conda
+    build_deps = ['cython', 'numpy x.x', 'swig']
```

Comment:
Should there also be a lint that we check numpy is specified as `numpy x.x` when it appears in the build requirements?